crates/core/src/client/config.rs

```diff
@@ -35,8 +34,8 @@ impl Network {
 
     pub fn provider_url(&self) -> Result<Url, ConfigError> {
         match self {
-            Network::Katana => Ok(Url::parse(KATANA_RPC_URL)?),
-            Network::Madara => Ok(Url::parse(MADARA_RPC_URL)?),
+            Network::Katana => Ok(Url::parse(std::env::var("KATANA_RPC_URL").unwrap().as_str())?),
+            Network::Madara => Ok(Url::parse(std::env::var("MADARA_RPC_URL").unwrap().as_str())?),
```

Comment:
Make the RPC URL configurable to be able to connect to containerized instances

Reply:
Can we avoid unwraps in this case?